docs/schema.graphql

```diff
@@ -1067,6 +1087,18 @@ enum OrderDirection {
   ASC
 }
 
+"""The origin of the deposit."""
+enum Origin {
```

Comment:
The documentation isn't right for these elements... Should be changed in the Sangria code, after which you should generate a new version of this file!